src/index.html

```diff
@@ -11,7 +11,133 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form method="POST" action="http://localhost:8080">
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="form_field">
+          Surname:
+          <input
+          type="text"
+          name="Surname"
+          autocomplete="off"
+          >
```

Comment:
```suggestion
          <input
            type="text"
            name="Surname"
            autocomplete="off"
          >
```
checklist p.3 https://github.com/mate-academy/layout_html-form/blob/master/checklist.md
fix it all over the page